### DISCUSSION TITLE
Add support for SOCKS5 proxies

### DISCUSSION THREAD

**Discussion by avivace** [Spotifyd/spotifyd#1319] [Ideas]
Currently, only HTTP proxy are supported and there's no way (to my knowledge) to make spotifyd work over an SSH proxy. Workaround also don't work, see https://github.com/Spotifyd/spotifyd/issues/1241

**Comment by eladyn** [comment]
Unfortunately, the different libraries we rely on don't support different proxy kinds. In particular, the proxy library that `librespot` uses, is quite restrictive in the types it can handle, IIRC. So we should probably work on the workaround issues, since we won't be able to easily fix our dependencies.

**Comment by eladyn** [comment]
I'm gonna convert this into a discussion, since it doesn't really seem actionable from our side.
